orchagent/orchdaemon.cpp

```diff
@@ -513,6 +516,17 @@ void OrchDaemon::start()
                 }
             }
         }
+        if (gIsSwitchShutdown) {
+            static time_t last_log = 0;
+
+            if ((time(NULL) - last_log) > SWITCH_SHUTDOWN_LOG_INTERVAL_IN_SECS) {
```

Comment:
Minor comment - orchagent files follow the style of having opening bracket `{` in a new line. Could you please change?

Reply:
@daall Wish this could be captured by linters instead of wasting @prsunny 's time.

Reply:
Done. Thanks Prince!